plugins/source/datadog/README.md

```diff
@@ -0,0 +1,35 @@
+# Datadog Plugin
```

Comment:
I think we'll need to move this to the Website, see example in https://github.com/cloudquery/cloudquery/tree/main/plugins/source/azure#readme
https://github.com/cloudquery/cloudquery/tree/0e4b8dc53358388f8a1e61cad8ae8a1ab2f52342/website/pages/docs/plugins/sources/azure

and changes in https://github.com/cloudquery/cloudquery/pull/5104